test/modules/core/shaderlib/project/project-glsl.spec.js

```diff
@@ -195,8 +190,8 @@ const TEST_CASES = [
         name: 'project_position',
         func: ({project_position}) => project_position([-122.05, 37.92, 0, 1], [0, 0]),
         output: getPixelOffset(
-          TEST_VIEWPORT_HIGH_ZOOM.projectFlat([-122.05, 37.92]),
-          TEST_VIEWPORT_HIGH_ZOOM.projectFlat([-122, 38])
+          TEST_VIEWPORT_HIGH_ZOOM.projectPosition([-122.05, 37.92, 0]),
```

Comment:
Did these need to be changed? Is the precision not supported in project flat? Or these are just better tests?

Reply:
`projectPosition` handles 3D points, while `projectFlat` (a legacy method) only handles x and y.